src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/Dashboard_TripSearchScreen.tsx

```diff
@@ -345,24 +348,29 @@ export const Dashboard_TripSearchScreen: React.FC<RootProps> = ({
                 viewContainerStyle={style.searchTimeButton}
               />
               {filtersState.enabled && (
-                <Button
-                  text={t(TripSearchTexts.filterButton.text)}
-                  accessibilityHint={t(TripSearchTexts.filterButton.a11yHint)}
-                  interactiveColor="interactive_0"
-                  mode="secondary"
-                  type="inline"
-                  compact={true}
-                  onPress={filtersState.openBottomSheet}
-                  testID="dashboardDateTimePicker"
-                  rightIcon={{
-                    svg: Filter,
-                    notification: filtersState.anyFiltersApplied
-                      ? {color: 'valid', backgroundColor: 'background_accent_0'}
-                      : undefined,
-                  }}
-                  viewContainerStyle={style.filterButton}
-                  ref={filtersState.onCloseFocusRef}
-                />
+                <View ref={filterButtonWrapper}>
```

Comment:
Just thinking out loud, not saying it should be changed in this PR:
Seeing that we now need a wrapper around the button to add another ref makes me think the bottom sheet close ref might be better to implement another way. Now the `useBottomSheet` hook creates the ref for the user, but it might be better that the user sends in the ref. That way we could just create a ref for a specific component, and that same ref could be used both as bottom sheet close ref and for popover target ref.

Reply:
That sounds like a good idea, though slightly outside the scope of this pr.